pages/download/[[os]].vue

```diff
@@ -22,15 +22,21 @@ definePageMeta({
   middleware: async (to) => {
     let downloadUrl = 'https://www.fosshub.com/JabRef.html'
     const os = to.params.os as string | undefined
-    if (os && ['win', 'mac', 'linux'].includes(os)) {
+    if (os && ['win_msi', 'win_zip', 'mac_arm64_dmg', 'mac_arm64_pkg', 'mac_x86_64_dmg', 'mac_x86_64_pkg', 'linux_deb', 'linux_rpm', 'linux_tar_gz'].includes(os)) {
       const { data } = await useFetch('/api/getLatestRelease')
```

Comment:
We are in the process of phasing out fosshub, can you change it to github release?

Reply:
I'll see what I could do+

Reply:
Hi @Siedlerchr, just finished migrating the links from Fosshub to GitHub Release. There have been troubles on my end though with the ```useFetch``` function for some reason that I'm not sure of; this hinders me from fully checking if the implementation's good or not. Using it at the moment in my branch yields an undefined value. I checked the response attained by the function and it said that I had bad credentials. Investigating further led me to see that I do not have a ```GITHUB_REPO_TOKEN``` attribute in my .env file. Should I perhaps put a personal GitHub token in the .env file for testing purposes? 

For now though, I'll try to work on the overview page. Much thanks!